crates/matrix-store-key/Cargo.toml

```
@@ -0,0 +1,20 @@
+[package]
+name = "matrix-store-key"
+version = "0.1.0"
+edition = "2021"
```

let's add the necessary or we'll have to do them later anyways:
```suggestion
edition = "2021"
description = "Helpers for encrypted storage keys for the Matrix SDK"
repo = "https://github.com/matrix-org/matrix-rust-sdk"
license = "Apache-2.0"
```